src/profiler.cpp

```diff
@@ -395,8 +395,6 @@ bool Profiler::start(Mode mode, int interval, int frame_buffer_size) {
     _frame_buffer_index = 0;
     _frame_buffer_overflow = false;
 
-    resetSymbols();
```

Comment:
Not sure how Java API (`javaApi.cpp`) is used, but it may break it if `VM::init` is not called.
Can revert this change and add `_native_lib_count == 0` check if so